### PR TITLE
Add some missing tests to TestParser2.

### PR DESCRIPTION
**Pull request by knz**
As suggested by @tamird and @petermattis in #4935.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4949)

### CODE REVIEW

**Comment by petermattis** [comment]
LGTM

File: sql/parser/parse_test.go [hunk 1]
```diff
@@ -475,6 +475,26 @@ func TestParse2(t *testing.T) {
 			`SELECT FROM t1 LEFT JOIN t2 ON a = b`},
 		{`SELECT FROM t1 RIGHT OUTER JOIN t2 ON a = b`,
 			`SELECT FROM t1 RIGHT JOIN t2 ON a = b`},
+		// Some functions are nearly keywords.
+		{`SELECT CURRENT_TIMESTAMP`,
+			`SELECT "CURRENT_TIMESTAMP"()`},
```
Comment (tamird) [inline sql/parser/parse_test.go]: whoa, is that right? why do we end up with the quotes?

**Comment by tamird** [comment]
LGTM, thanks for this.

File: sql/parser/parse_test.go (hunk 1 above)
Comment (knz) [inline sql/parser/parse_test.go]: That happens whenever we have a keyword as function name.

**Pull request merged**